File: adu/CP/Code/DTTin/4.py
t = int(input())
for i in range(t):
    cnt = 0
    n = int(input())
    while n!=0:
        if n%2==1:
            cnt = cnt+1
        n=int(n/2)
    if cnt%2==0:
        print("even")
    else: print("odd")

File: adu/CP/Code/DTTin/4.java
import java.util.*;
class Main{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int t = scan.nextInt();
        for (int i=0; i<t; i++){
            int cnt = 0;
            long n = scan.nextLong();
            while(n!=0){
                if(n%2==1) cnt++;
                n/=2;
            }
            if(cnt%2==0)
                System.out.println("even");
            else System.out.println("odd");
        }
    }
}